packages/node_modules/@ciscospark/internal-plugin-ediscovery/src/report-generator.js

```diff
@@ -0,0 +1,133 @@
+import SparkPlugin from '@ciscospark/spark-core';
+import JSZip from 'jszip';
+import saveAs from 'file-saver';
+import MimeBuilder from 'emailjs-mime-builder';
+
+/**
+ * @class ReportGenerator is used to compile, zip, and save an eDiscovery report to disk.
+ *
+ * Warning: This file is browser specific. It cannot function in a NodeJS server environment.
+ */
+const ReportGenerator = SparkPlugin.extend({
+  namespace: 'Ediscovery',
+
+  props: {
+    reportId: 'string',
+    zip: 'object',
+    folder: 'object',
+    spaces: 'object'
+  },
+
```

Comment:
Activities are currently mapped to a mime format and stored as EML files which are downloaded to the users hard disk.